cartography/intel/aws/ec2/route_tables.py

```diff
@@ -26,7 +27,17 @@ def get_route_tables_data(boto3_session: boto3.session.Session, region: str) ->
         route_tables: List[Dict] = []
         for page in paginator.paginate():
             route_tables.extend(page['RouteTables'])
+
+        default_vpc = get_default_vpc(client)
+
         for route_table in route_tables:
+            if route_table.get('VpcId') == default_vpc.get('VpcId'):
```

Comment:
what if i add an entry on my own. how do we differentiate in that case.

Reply:
For this what I have did is:
1. We now mark everything as 'user' by default
2. We only mark a route table as 'predefined' if:
    - It belongs to the default VPC (By matching the VPC IDs) and
    - It has an association with `Main: true`